modules/features/player/src/test/java/au/com/shiftyjelly/pocketcasts/player/util/MainCoroutineRule.kt

```diff
@@ -0,0 +1,24 @@
+package au.com.shiftyjelly.pocketcasts.player.util
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestDispatcher
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+
+@ExperimentalCoroutinesApi
+class MainCoroutineRule(
```

Comment:
This is duplicated across modules and needs to be extracted to a common test module or a test fixture if possible.